ih/palettes/floss.txt

```diff
@@ -1,108 +1,254 @@
+150,#740733
```

Comment:
What are these additions? Are you merging in fullfloss.txt_? If so what is floss2.txt?

(I may be tempted to add comment handling to these files if that'd help things)

Reply:
That's right!

- Reset floss.txt to fullfloss.txt -- from the git history I think floss.txt was originally fullfloss.txt_ but due to palette truncation wasn't working properly
- floss2.txt is a different floss -> hex chart I cribbed off the JS of http://threadcolors.com/ and http://www.damaniel.info/dmc/dmctorgb.html (they use the same dictionary, I learned, after painstakingly converting them to this format and then diffing :joy:)